ui/src/organizations/components/Variables.tsx

```diff
@@ -35,21 +36,50 @@ interface Props {
 
 interface State {
   searchTerm: string
-  overlayState: OverlayState
+  createOverlayState: OverlayState
+  importOverlayState: OverlayState
 }
 
 export default class Variables extends PureComponent<Props, State> {
+  private get emptyState(): JSX.Element {
+    const {orgName} = this.props
+    const {searchTerm} = this.state
+
+    if (_.isEmpty(searchTerm)) {
+      return (
+        <EmptyState size={ComponentSize.Medium}>
+          <EmptyState.Text
+            text={`${orgName} does not own any Variables , why not create one?`}
```

Comment:
is the ` , ` necessary?

Reply:
I think it adds clarity to have some punctuation there